src/NUnitFramework/framework/Internal/Tests/TestMethod.cs

```diff
@@ -135,7 +142,7 @@ public override TNode AddToXml(TNode parentNode, bool recursive)
         /// <value>A list of child tests</value>
         public override IList<ITest> Tests
         {
-            get { return new ITest[0]; }
+            get { return _emptyArray; }
```

Comment:
When filtering a large number of tests this allocates a lot of throwaway objects.

Reply:
Makes sense 👍